src/tester.rs

```diff
@@ -37,52 +38,128 @@ pub fn run_test(run: &mut Command, testcase: &ClashTestCase) -> Result<TestRunRe
 
     let output = run.wait_with_output()?;
     let stdout = String::from_utf8(output.stdout)?;
-    let stdout = stdout.trim_end().to_string();
+    let mut stdout = stdout.trim_end().to_string();
+    // Fixes Windows "\r\n" endings not returning Success when they should
+    stdout = stdout.replace("\r", "");
```

Comment:
this should be `.replace("\r\n", "\n")` (if someone's code prints '\r' in the middle of the line we don't want to replace that)